example-workflows/review-app.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
   review_app:
     runs-on: ubuntu-latest
     # only run when a pull request is opened
-    if: github.event_name == 'pull_request' && github.event.action == 'opened'
```

Comment:
Thinking about this more, this should only run when the review app was created, since thats when we call `apps:clone`. Instead, this job should be named `review_app_create` and then we have a `review_app_push` job that runs when the action is not opened or closed, which then deploys the app.